src/UI/Art/SpriteArtist/SpriteLibrary.java

```diff
@@ -358,6 +359,15 @@ public static Sprite getMapUnitLetterSprites()
     return mapUnitLetterSprites;
   }
 
+  public static Sprite getMapUnitSymbolSprites()
+  {
+    if( null == mapUnitSymbolSprites )
+    {
+      mapUnitSymbolSprites = new Sprite(SpriteLibrary.createDefaultBlankSprite(8, 8), 8, 8);
```

Comment:
Can't say I'm a super fan of this.

Reply:
Aww, I was hoping you wouldn't notice.

Whenever we have a need for those sprites, we can add 'em. Until then, they will just show up as blank spaces (just like unsupported characters in the other fonts. This is, in any case, not a regression in functionality.

Reply:
I say try to load the resource, and fail as normal if it's not there (which it won't be).
At least that way, you get a warning and a black rectangle.

Reply:
I might rather have an empty space than a black rectangle. I could just put the resource there as a blank sprite I guess.

Not that big a deal either way, since we aren't using any of those characters yet. Whatever you prefer.

Reply:
I would rather have the loading logic in place, rather than a cleverly-disguised no-op.
Whether you add a resource to load is your choice.